packages/flutter/lib/src/widgets/scroll_simulation.dart

```diff
@@ -135,8 +135,8 @@ class ClampingScrollSimulation extends Simulation {
     @required this.velocity,
     this.friction: 0.015,
     Tolerance tolerance: Tolerance.defaultTolerance,
-  }) : super(tolerance: tolerance) {
-    assert(_flingVelocityPenetration(0.0) == _kInitialVelocityPenetration);
+  }) : assert(_flingVelocityPenetration(0.0) == _kInitialVelocityPenetration),
```

Comment:
`_flingVelocityPenetration` is a static function and the content of this `assert` has always the same result. It could be turned into a test case to insure `_flingVelocityPenetration(0.0) == _kInitialVelocityPenetration`.